lib/tasks/flag_reservations.rake

```diff
@@ -0,0 +1,43 @@
+namespace :flag_reservations do
+  desc 'Flag reservations due yesterday as overdue'
+  task flag_overdue: :environment do
+    new_overdue = Reservation.where('due_date = ?',
+                                    Time.zone.yesterday).checked_out
```

Comment:
I prefer using `Time.zone.today - 1.day` just so that we're consistent with our references (here and in the other tasks below).